pkg/client/cli/cliutil/exec_unix.go

```diff
@@ -0,0 +1,74 @@
+// +build !windows
+
+package cliutil
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+
+	//nolint:depguard // Because we won't ever .Wait() for the process and we'd turn off
+	// logging, using dexec would just be extra overhead.
+	"os/exec"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/datawire/dlib/dcontext"
+	"github.com/datawire/dlib/dexec"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/logging"
+	"github.com/telepresenceio/telepresence/v2/pkg/proc"
+)
+
+func background(ctx context.Context, exe string, args []string) error {
+	// The context should not kill it if cancelled
+	ctx = dcontext.WithoutCancel(ctx)
+	cmd := exec.CommandContext(ctx, exe, args...)
```

Comment:
If you're using stdlib `exec` instead of dlib `dexec`, the *only* thing that the ctx given to `CommanContext` does is cancel it, so if you're using `WithoutCancel`, might as well just use `dexec.Command` (no Context).